systemtests/tests/testfind/etc/bareos/bareos-sd.d/device/FileStorage.conf

```diff
@@ -0,0 +1,15 @@
+Device {
```

Comment:
Is the bareos-sd.d directory required?

Reply:
it is not required, new this after testing